pswalker/iterwalk.py

```diff
@@ -114,6 +114,13 @@ def iterwalk(detectors, motors, goals, starts=None, first_steps=1,
     filters: list of dictionaries
         Each entry in this list should be a valid input to the filters argument
         in the lower functions, such as walk_to_pixel and measure_average.
+    
+    tol_scaling: list of floats or Nones, optional
+        Constants for adaptive tolerences. Nones or omitting this argument
+        reverts to fixed tolerence. For early walks with a starting point far
+        from the goal, tolerence for the walk is set at 
+        current_dist/tol_scaling instead of the set tolerence. Scaling ends
+        when calculated tolerence < targeted tolerence.  
```

Comment:
Could you also make sure tol_scaling gets passed into the recovery plan? The docstring above says that all the arguments to iterwalk are passed to the recovery plan.